lib/cc/cli/engines/remove.rb

```diff
@@ -7,13 +7,12 @@ class Remove < EngineCommand
         def run
           require_codeclimate_yml
 
-          if !engine_exists?
+          if !valid_engine?
             say "Engine not found. Run 'codeclimate engines:list' for a list of valid engines."
```

Comment:
Had the same thought and no strong opinion so left it as is. Fatal makes sense to me.